pyalex/api.py

```diff
@@ -169,8 +169,12 @@ def __init__(self, params=None):
     def _get_multi_items(self, record_list):
         return self.filter(openalex_id="|".join(record_list)).get()
 
-    def _full_collection_name(self):
-        return config.openalex_url + "/" + self.__class__.__name__.lower()
+    def _full_collection_name(self, autocomplete=False):
+        if autocomplete:
+            base_url = config.openalex_url + "/autocomplete/"
+            return base_url + self.__class__.__name__.lower()
+        else:
+            return config.openalex_url + "/" + self.__class__.__name__.lower()
```

Comment:
I think we can simplify this and keep it unchanged.

Reply:
We could replace `if autocomplete:` by `if 'q' in self.params.keys():`